cmd/keeper/keeper.go

```diff
@@ -1039,18 +1039,35 @@ func (p *PostgresKeeper) resyncIfNotReady(followed *cluster.KeeperState, initial
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
 	if !ready {
-		log.Info("Standby is not accepting connections. It's probably waiting for unavailable WALs. Forcing a full resync.")
-		if err = p.resync(followed, initialized, started); err != nil {
-			return trace.Wrap(err, "failed to full resync from followed instance")
+		log.Info("Standby is not accepting connections. Forcing a full resync.")
+		if err = p.resyncAndStart(followed, initialized, started); err != nil {
+			return trace.Wrap(err)
 		}
-		if err = p.pgm.Start(); err != nil {
-			return trace.Wrap(err, "error starting PostgreSQL instance")
+	}
+
+	if err = p.pgm.IsStreaming(); err != nil {
+		log.Info("Standby is not accepting connections. It's probably waiting for unavailable WALs. Forcing a full resync.")
+		log.Debugf("Error: %v", err)
```

Comment:
Is this going to spam logs if you logged this as `Warn`? How interesting would be this error for possible troubleshooting?

Reply:
This line supposed to be deleted. Left it after debugging the code.